website/src/libs/api-v3/errors.js

```diff
@@ -7,8 +7,17 @@ export class CustomError extends Error {
   }
 }
 
-// NotAuthorized error with a 401 http error code
-// used when a request is not authorized
+/**
```

Comment:
errors for sure, but I think you can also define permission and in general every function/class can be defined once and used in multiple places